docs/03-tutorials/00-serverless/svls-11-set-asynchronous-connection-of-functions.md

```diff
@@ -111,7 +109,14 @@ This tutorial shows only one possible use case. There are many more use cases on
    The `init` command creates the same files as in the `emitter` folder.
 
 2. In the `config.yaml` file, configure event types your Function will subscribe to:
-    ```yaml
+
+<div tabs name="function" group="set-asynchronous-connection-of-functions">
+  <details>
+  <summary label="v1alpha1">
+  v1alpha1
```

Comment:
this field should contain `v1` instead of `v1alpha1` it's the specification version, not the resource version

```suggestion
  <summary label="v0">
  v0
```